src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-details/rgw-storage-class-details.component.html

```diff
@@ -234,6 +173,19 @@
             <td>{{ selection?.multipart_sync_threshold | dimlessBinary }}</td>
           </tr>
         }
+          <tr>
+            <td class="bold"
+                i18n>
+              Placement Target
```

Comment:
```suggestion
              >
              <span i18n>Placement Target<span>
```
Always try to extrapolate with closest , for demonstration used span, but you can use whatever suites,